internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -1003,39 +993,48 @@ func (s *Server) processPullRequestEvent(
 		return nil, fmt.Errorf("error creating pull request properties: %w", err)
 	}
 
-	provider, err := s.providerManager.InstantiateFromID(ctx, repoEnt.Entity.ProviderID)
+	repoProps, err := properties.NewProperties(map[string]any{
+		properties.PropertyUpstreamID: strconv.FormatInt(ghRepo.GetID(), 10),
+		properties.PropertyName:       ghRepo.GetName(),
```

Comment:
is the name strictly needed? can we do just with the upstream ID?

Reply:
done in a separate patch for easier re-review